internal/services/machinelearning/machine_learning_compute_cluster_resource_test.go

```diff
@@ -366,7 +366,11 @@ resource "azurerm_machine_learning_compute_cluster" "test" {
 func (r ComputeClusterResource) template_basic(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
```

Comment:
this highlights there's a bug in the resource which needs to be fixed as something isn't being correctly deleted, what's the error here?